src/components/DataSession/OperationWizard.vue

```diff
@@ -101,23 +103,42 @@ function goForward() {
       }
     }
     for (const inputKey in selectedImages.value) {
-      let selected = []
+      let input = []
+      const filter = selectedOperationInputs.value[inputKey].filter
       selectedImages.value[inputKey].forEach(index => {
-        selected.push(props.images[index])
+        input.push(imagesWithFilter(filter)[index])
       })
-      operationDefinition.input_data[inputKey] = selected
+      operationDefinition.input_data[inputKey] = input
     }
     emit('addOperation', operationDefinition)
     emit('closeWizard')
   }
 }
 
+function imagesWithFilter(filters) {
```

Comment:
this is just a suggestion for a better UX (imo and doesn't have to be implemented now but to consider in the future): instead of getting a warning, could the user just not select that operation if none of the images qualify for the rgb filter?

Reply:
done!